firstpartydata/tests/extractbidderconfigfpd/valid-fpd-bidder-specific-only.json

```diff
@@ -0,0 +1,109 @@
+{
+  "inputRequestData": {
+    "data": {},
+    "bidderconfig": [
+      {
+        "bidders": [
+          "appnexus"
+        ],
+        "config": {
+          "ortb2": {
+            "site": {
+              "id": "apnSiteId",
+              "ext": {
+                "data": {
+                  "sitefpddata": "sitefpddata",
+                  "moreFpd": {
+                    "fpd": 123
+                  },
+                  "morefpdData": "morefpddata"
+                }
+              }
+            }
+          }
+        }
+      },
+      {
+        "bidders": [
```

Comment:
I'd say having bidder config for a single bidder for this test case is enough as we're trying to test that when just bidder specific FPD is provided, we pass in only that to the bidders.

In general, for all test cases, I'd recommend we keep the data in the test cases as minimum as possible to test that scenario. This helps make the test cases much easier to read and follow. And then you can have a few (perhaps 3-5) really complicated test cases that touch all the different scenarios.